### PR TITLE
ADCM-2503 Tests on running actions with permissions and rbac tests directory refactoring

### PR DESCRIPTION
**Pull request by Sealwing**

### CODE REVIEW

**Review by annademina — APPROVED** [review]

File: tests/functional/rbac/actions/test_run_actions.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+
+def _do_nothing_action_not_presented(obj):
+    """Filter for removing objects that doesn't have "Do nothing" action"""
+    return DO_NOTHING_ACTION not in (a.display_name for a in obj.action_list())
+
+
+# !===== Tests ======!
+
+# pylint: disable-next=too-many-locals
```
Comment (skhomuti) [inline tests/functional/rbac/actions/test_run_actions.py:42]: are you sure it's needed here?

File: tests/functional/rbac/actions/test_run_actions.py [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        f'Check that granting policy allows to run action "{DO_NOTHING_ACTION}" on {get_object_represent(adcm_object)}'
+    ):
+        (user_object,) = as_user_objects(user_sdk, adcm_object)
+        business_role = action_business_role(adcm_object, DO_NOTHING_ACTION)
+        policy = create_action_policy(admin_sdk, adcm_object, business_role, user=user)
+
+        check_single_action_is_allowed_on_object(DO_NOTHING_ACTION, adcm_object, user_sdk, business_role)
+
+    with allure.step(f"Check that granted permission doesn't allow running '{DO_NOTHING_ACTION}' on other objects"):
+        with user_objects_map(
```
Comment (skhomuti) [inline tests/functional/rbac/actions/test_run_actions.py:87]: maybe check actions explicitly without too complex `user_objects_map` func?

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: tests/functional/rbac/actions/test_run_actions.py (hunk 1 above)
Comment (Sealwing) [reply tests/functional/rbac/actions/test_run_actions.py:42]: Nope, it's redundant now, thanks

Comment (skhomuti) [reply tests/functional/rbac/actions/test_run_actions.py:42]: and below too

File: tests/functional/rbac/actions/test_run_actions.py (hunk 2 above)
Comment (Sealwing) [reply tests/functional/rbac/actions/test_run_actions.py:87]: At first, I was thinking about `user_objects_map` as a function to be reused to filter different kinds of "denied" objects, but I'm still not sure that it'll be somehow different without this func. I may unwrap it inside the `check_action_is_not_allowed_on_objects`, but do you think it's worth it?

File: tests/functional/rbac/actions/test_run_actions.py (hunk 1 above)
Comment (Sealwing) [reply tests/functional/rbac/actions/test_run_actions.py:42]: yep

**Review by skhomuti — APPROVED** [review]

**Pull request merged**